pkg/api/dataproxy_test.go

```diff
@@ -28,7 +29,7 @@ func TestDataProxy(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
-			assert.Equal(t, tc.exp, ensureProxyPathTrailingSlash(tc.origPath, tc.proxyPath))
+			assert.Equal(t, tc.exp, datasourceproxy.EnsureProxyPathTrailingSlash(tc.origPath, tc.proxyPath))
```

Comment:
This test should be located in the datasourceproxy package/where the function is defined/used